GOVERNANCE.md

```diff
@@ -1,38 +1,38 @@
-# OpenJS Standards Working Group
+# OpenJS Standards Collaboration Space
 
 <!-- yet to add -->
-For the current list of the Working Group members, see the project [README.md](./README.md).
+For the current list of the Collaboration Space members, see the project [README.md](./README.md).
 
 ## Members
 
 The [openjs-foundation/standards](https://github.com/openjs-foundation/standards) GitHub
-repository is maintained by the Working Group and additional Members who are
+repository is maintained by the Collaboration Space and additional Members who are
 added on an ongoing basis.
 
 * Invited to all meetings
 * Can participate in [consensus seeking process](#consensus-seeking-process)
-* Counted towards quorum in [Working Group Meetings](#working-group-meetings)
+* Counted towards quorum in [Collaboration Space Meetings](#collaboration-space-meetings)
 * Participates in voting
 
-## Working Group Membership
+## Collaboration Space Membership
```

Comment:
this is going to break links; we should probably add an `<a id="working-group-membership"></a>` and similar throughout to preserve them

Reply:
well, technically going to break fragments, which is not nearly as bad.  I'd rather let them break than add html to md  🤷

Reply:
fragments are still part of a URL :-/

Reply:
I just checked and there are no links to this in the openjs org's repos.  are there links elsewhere that you know of?

Reply:
nope - but the potential exists. just like semver is about theoretical breakage not actual breakage, URL preservation is about potential breakage not just known breakage.

Reply:
I find this semi-breakage an acceptable sacrifice